lib/lveditqmljs/src/codeqmlhandler.cpp

```diff
@@ -465,25 +480,7 @@ void CodeQmlHandler::setDocument(ProjectDocument *document){
     m_highlighter->setTarget(m_target);
     d->documentScope = DocumentQmlScope::createEmptyScope(d->projectHandler->scanMonitor()->projectScope());
 
-    if ( m_document ){
```

Comment:
I saw the deletion of `QmlEditFragment`s was moved someplace else (line 945, inside `aboutToDelete`), but I didn't manage to find the connections to the `contentsChange`, `cursorPositionChanged` and to the `formatChanged`. Were they moved or do we have to add them back?